.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: go
 go:
   - "1.x"
-  - "1.9"
-  - "1.10"
```

Comment:
Could we keep 1.10 just to prove it still works without go modules?

Reply:
I'd like to, but getting this error in ci:
https://travis-ci.org/JumboInteractiveLimited/Gandalf/jobs/531592820
```
$ go get -t ./...
package github.com/jmartin82/mmock/definition: cannot find package "github.com/jmartin82/mmock/definition" in any of:
	/home/travis/.gimme/versions/go1.10.linux.amd64/src/github.com/jmartin82/mmock/definition (from $GOROOT)
	/home/travis/gopath/src/github.com/jmartin82/mmock/definition (from $GOPATH)
```
Which is the reason why I'm updating to use modules, as it should be a quick fix.
Do you know of an easy way to get a specific version of a repo using go get?

Reply:
I appreciate the feedback!

Reply:
ah I see, sadly not without modules, if we have to drop 1.10 thats fine, was just a hope :+1: